### PR TITLE
Update to openSUSE-15.3

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
fixes https://github.com/SUSE/spacewalk/issues/15042

### CODE REVIEW

File: jenkins_pipelines/environments/uyuni-prs-ci-tests [hunk 1]
```diff
@@ -19,7 +19,7 @@ node('pull-request-test') {
             string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/uyuni.git', description: 'Change this by your repo, only if you changed the tests in your PR'),
             string(name: 'cucumber_ref', defaultValue: 'master', description: 'Change this by your branch, only if you changed the tests in your PR'),
             string(name: 'email_to', defaultValue: '', description: 'Receive a report to your e-mail when it finish'),
-            string(name: 'build_repo', defaultValue: 'openSUSE_Leap_15.2', description: 'Repository to build the packages for'),
+            string(name: 'build_repo', defaultValue: 'openSUSE_Leap_15.3', description: 'Repository to build the packages for'),
```
Comment (juliogonzalez) [inline jenkins_pipelines/environments/uyuni-prs-ci-tests:22]: I am confused, why don't you use this value for the URLs? That would make maintenance easier.

**Review by juliogonzalez — COMMENTED** [review]
LGTM, but one question

File: jenkins_pipelines/environments/common/pipeline-pull-request.groovy [hunk 2]
```diff
@@ -71,9 +71,9 @@ def run(params) {
...
                             if(params.publish_in_host) {
-                              echo "Publishing packages into http://${fqdn_jenkins_node}/workspace/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.2/x86_64"
-                              sh "bash -c \"rm -rf ${jenkins_workspace}/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.2/x86_64\""
-                              sh "bash susemanager-utils/testing/automation/publish-rpms.sh -p \"${params.builder_project}:${params.pull_request_number}\" -r openSUSE_Leap_15.2 -a x86_64 -d \"${jenkins_workspace}\""
+                              echo "Publishing packages into http://${fqdn_jenkins_node}/workspace/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.3/x86_64"
+                              sh "bash -c \"rm -rf ${jenkins_workspace}/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.3/x86_64\""
+                              sh "bash susemanager-utils/testing/automation/publish-rpms.sh -p \"${params.builder_project}:${params.pull_request_number}\" -r openSUSE_Leap_15.3 -a x86_64 -d \"${jenkins_workspace}\""
```
Comment (srbarrios) [inline jenkins_pipelines/environments/common/pipeline-pull-request.groovy:76]: We might want to replace the hardcoded  openSUSE_Leap_15.3 by `${params.build_repo}`

**Review by srbarrios — APPROVED** [review]

File: jenkins_pipelines/environments/uyuni-prs-ci-tests (hunk 1 above)
Comment (jordimassaguerpla) [reply jenkins_pipelines/environments/uyuni-prs-ci-tests:22]: that is a good idea. Let me change it and push it again.

File: jenkins_pipelines/environments/common/pipeline-pull-request.groovy (hunk 2 above)
Comment (jordimassaguerpla) [reply jenkins_pipelines/environments/common/pipeline-pull-request.groovy:76]: Yes, good idea, the same julio is suggesting. Let me do it and I'll push it.

File: jenkins_pipelines/environments/uyuni-prs-ci-tests (hunk 1 above)
Comment (jordimassaguerpla) [reply jenkins_pipelines/environments/uyuni-prs-ci-tests:22]: done

File: jenkins_pipelines/environments/common/pipeline-pull-request.groovy (hunk 2 above)
Comment (jordimassaguerpla) [reply jenkins_pipelines/environments/common/pipeline-pull-request.groovy]: done

**Pull request merged**